src/ocm/components/clusterConfiguration/review/ReviewNetworkingTable.tsx

```diff
@@ -49,26 +49,30 @@ export const ReviewNetworkingTable = ({ cluster }: { cluster: Cluster }) => {
           isDualStack(cluster) && { title: 'Primary' },
         ],
       },
-      {
-        rowId: 'api-ip',
-        cells: [
-          { title: 'API IP' },
-          {
-            title: cluster.apiVip,
-            props: { 'data-testid': 'api-vip', colSpan: 2 },
-          },
-        ],
-      },
-      {
-        rowId: 'ingress-ip',
-        cells: [
-          { title: 'Ingress IP' },
-          {
-            title: cluster.ingressVip,
-            props: { 'data-testid': 'ingress-vip', colSpan: 2 },
-          },
-        ],
-      },
+      cluster.apiVip
+        ? {
+            rowId: 'api-ip',
+            cells: [
+              { title: 'API IP' },
+              {
+                title: cluster.apiVip,
+                props: { 'data-testid': 'api-vip', colSpan: 2 },
+              },
+            ],
+          }
+        : '',
+      cluster.ingressVip
+        ? {
+            rowId: 'ingress-ip',
+            cells: [
+              { title: 'Ingress IP' },
+              {
+                title: cluster.ingressVip,
+                props: { 'data-testid': 'ingress-vip', colSpan: 2 },
+              },
+            ],
+          }
+        : '',
     ];
```

Comment:
```suggestion
        : '',
    ];.filter(Boolean)
```

Reply:
I have a bad feeling about this...

Reply:
`.filter(Boolean)` is not the way. But pushing with a condition works. We can go that route.